src/test/java/uk/gov/cslearning/record/service/booking/BookingNotificationServiceTest.java

```diff
@@ -0,0 +1,121 @@
+package uk.gov.cslearning.record.service.booking;
```

Comment:
Do you have any negative scenario to test in this service? If yes, then it would be good to cover those branches as well.

Reply:
we don't have negative scenarios in this case as a booking has been made and if something goes wrong then the user/line manager will not get an email. So, the notifying stage in the last stage after all the checks has been done.